File: dev-docs/snippets/databases/remove-manager.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Remove a manager from a database.
export async function removeManager(
  client: JsBaoClient,
  databaseId: string,
  userId: string,
) {
  // #region example
  const result = await client.databases.removeManager(databaseId, userId);
  // #endregion example
  return result;
}

File: dev-docs/snippets/databases/remove-manager.swift
import JsBaoClient

// Remove a manager from a database.
func removeManager(client: JsBaoClient, databaseId: String, userId: String) async throws {
  // #region example
  let result = try await client.databases.removeManager(
    databaseId: databaseId,
    userId: userId
  )
  // #endregion example
  _ = result
}
